packages/driver/src/cross-origin/patches/cookies.js

```diff
@@ -1,3 +1,46 @@
+const parseCookieString = (cookieString) => {
```

Comment:
This file is going to be interesting to convert into an injection time patch when the spec bridge isn't attached yet.